cxx-squid/src/main/java/org/sonar/cxx/CxxMetricsFactory.java

```diff
@@ -52,7 +52,8 @@ private CxxMetricsFactory() {
     // Introduce metric keys for sensors - number of issues per file / per module
     CLANG_SA_SENSOR_ISSUES_KEY("ClangSA"),
     CLANG_TIDY_SENSOR_ISSUES_KEY("Clang-Tidy"),
-    COMPILER_SENSOR_ISSUES_KEY("Compiler"),
+    VC_SENSOR_ISSUES_KEY("Compiler-VC"),

```

Comment:
Which metric keys should we use?
Compiler-XXX, or only XXX?

Reply:
Compiler-XXX look more consistent to me